### DISCUSSION TITLE
Filter effect: Vignette that follows video aspect ratio

### DISCUSSION THREAD

**Discussion by Grooves1210** [smplayer-dev/smplayer#798] [Q&A] [answered]
I'm using a vignette filter effect:

in Preferences -->  Advanced --->  MPlayer/mpv ---> Video Filters
`vignette=angle=1.3:aspect=1.4`

Instead of a fixed `aspect` I'd like it to automatically follow the videos aspect ratio.  eg width / height

What command can I use?   Thanks

**Comment by Raza-0007** [comment, marked as answer]
mpv does not have a native vignette filter. [See this article](https://github.com/mpv-player/mpv/blob/master/DOCS/man/vf.rst).

The vignette filter used by mpv is the one [implemented by  ffmpeg](https://ffmpeg.org/ffmpeg-filters.html#vignette-1). If you click the link you will find all the options and parameters associated with this filter, and it does not appear to have an option or a command to do what you need.

~Raza

**Reply by Grooves1210** [reply]
OK, thanks!
